Core/clim-basic/text-selection.lisp

```diff
@@ -52,46 +52,6 @@
 
 (defparameter *marking-border* 1)
 
-;;;; Text Selection Protocol
-
-(defgeneric release-selection (port &optional time)
-  (:documentation "Relinquish ownership of the selection."))
-
-(defgeneric request-selection (port requestor time)
-  (:documentation
-   #.(format nil "Request that the window system retrieve the selection ~
-                  from its current owner. This should cause a ~
-                  selection-notify-event to be delivered.")))
-
-(defgeneric bind-selection (port window &optional time)
-  (:documentation "Take ownership of the selection."))
-
-(defgeneric send-selection (port request-event string)
-  (:documentation "Send 'string' to a client in response to a selection-request-event."))
-
-(defgeneric get-selection-from-event (port event)
-  (:documentation "Given a selection-notify event, return a string containing
-the incoming selection."))
-
-;;; These events are probably very X11 specific.
-
-;;; Backends will likely produce subclasses of selection-notify-event
-;;; and selection-request-event.
-
-(defclass selection-event (window-event)
-  ((selection :initarg :selection
-              :reader selection-event-selection)))
-
-(defclass selection-clear-event (selection-event)  ())
-(defclass selection-notify-event (selection-event) ())
-(defclass selection-request-event (selection-event)
-  ((requestor :initarg :requestor :reader selection-event-requestor)))
-
-
```

Comment:
there are remaining implementations of these protocols in codebase (i.e Null backend). please grep for them and remove them. Also, if you remove protocols please remove them from documentation too.